projects/1.20/assets/oceanic-delight/oceanic_delight/lang/zh_cn.json

```
@@ -0,0 +1,74 @@
+{
+  "advancements.hitting_villager_with_net.descr": "用渔网攻击一名村民",
+  "advancements.hitting_villager_with_net.title": "为什么打我？！好痛！",
```

```suggestion
  "advancements.hitting_villager_with_net.title": "“为什么打我？！好痛！”",
```

我不知道为什么原文只有一侧引号，总是建议是这么走